rules/Php72/NodeFactory/AnonymousFunctionFactory.php

```diff
@@ -169,6 +169,14 @@ private function createUseVariablesFromParams(array $nodes, array $params): arra
                 continue;
             }
 
+            // Superglobal variables cannot be in a use statement
+            if (\in_array(
+                $variableName,
+                ['GLOBALS', '_SERVER', '_GET', '_POST', '_FILES', '_REQUEST', '_SESSION', '_ENV', '_COOKIE']
+            )) {
+                continue;
+            }
```

Comment:
you should can use `ReservedKeywordAnalyzer` service

https://github.com/rectorphp/rector-src/blob/6307286c278beedcb8ca379ff31d73cf31628544/src/Php/ReservedKeywordAnalyzer.php#L24

Reply:
Brilliant, thank you! I will make the update